malaria24/ona/models.py

```diff
@@ -74,13 +76,42 @@ class ReportedCase(models.Model):
     digest = models.ForeignKey('Digest', null=True, blank=True)
     ehps = models.ManyToManyField('Actor', blank=True)
 
+    def get_facilities(self):
+        return Facility.objects.filter(facility_code=self.facility_code)
+
+    def get_facility_attributes(self, attname):
+        return ', '.join([
+            getattr(f, attname)
+            for f in self.get_facilities() if getattr(f, attname)
+        ]) or "Unknown"
+
+    @property
+    def case_number(self):
+        facilities = self.get_facilities()
+        if facilities.count() > 0:
+            logging.warning('Multiple facilities found for case %s' % (
+                self.pk,))
```

Comment:
That whole block's now been removed as I'm now doing the case number generation in the jsbox app.